opacus/tests/grad_samples/linear_test.py

```diff
@@ -27,9 +28,10 @@ def test_input_bias(
         H: int,
         input_dim: int,
         bias: bool,
+        batch_first: bool,
     ):
 
-        if input_dim == 2:
+        if input_dim == 2:  # TODO: fix when dim is 2
```

Comment:
I don't understand: I think `batch_first` flag should have an effect on `x_shape`

Reply:
So I just changed the tests (line 18) to check `input_dim = 3` and `4`.